src/components/AnonymousChat/MessageChannelItem.tsx

```diff
@@ -49,6 +50,16 @@ const MessageChannelItem: (props: MessageChannelItemProps) => React.ReactElement
       }
     : null;
 
+  const onFollowUser = () => {
+    followUserAction(item);
+  };
+
+  React.useEffect(() => {
+    if (!dbAnonUserInfo && item?.channelType === 'PM') {
+      getOtherProfileFollowStatus(item?.user?.id);
+    }
+  }, []);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This useEffect hook is used to fetch the follow status of other users when the channel type is 'PM'. It's important to note that the dependency array is empty, which means this effect will only run once after the initial render. If you want this effect to run whenever `dbAnonUserInfo` or `item` changes, you should include these in the dependency array.
```javascript
React.useEffect(() => {
  if (!dbAnonUserInfo && item?.channelType === 'PM') {
    getOtherProfileFollowStatus(item?.user?.id);
  }
}, []);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->